### ISSUE TITLE
Feature request | Save Windows single desktop instead of all

### ISSUE THREAD

**Issue by ph00lt0** [alyssaxuu/later#1]
It would be great if the app allowed to save individual desktops, allowing to have multiple windows open from f.x. a browser/code editor and being able hide/reveal a desktop on demand to avoid cluttering up the app switching menu's

**Comment by WrathOP** [comment]
Yes this feature is what I thought this app will do but its seems its like when you click reopen current windows after reboot while rebooting , it just opens every app at you at once on the same desktop and calls it a day
